From a9ef93b2384aee9fa89d51b7ae5d744964c36b28 Mon Sep 17 00:00:00 2001
From: bluew <bluewww@users.noreply.github.com>
Date: Wed, 18 Nov 2020 14:14:59 +0100
Subject: [PATCH] reggen: Fix wrong type emitted in auto generated params

---
 validate.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/validate.py b/validate.py
index e48b796..849c4bf 100644
--- a/validate.py
+++ b/validate.py
@@ -48,7 +48,7 @@ def check_count(top, mreg, err_prefix):
         top["param_list"].append({
             "name": mreg["name"],
             "type": "int",
-            "default": mcount,
+            "default": str(mcount),
             "desc": "auto added parameter",
             "local": "true"
         })
-- 
2.29.0

